### PR TITLE
Display red badge after 3 days

### LINKED ISSUE #2837: Red badge: display it 3 days after the plugin activation
**Description:** showing the red badge 7 days after the plugin activation is not having the impact that we were expecting (#2071). See the results here: p1626887687077600-slack-wcpay-growth
We're still trying to find a good balance between taking advantage of the red badge effectiveness to increase the focus on WCPayments without distracting users or hurting the WCPayments activation.

**Acceptance criteria:**

- [ ] Start showing the red badge 3 days after the plugin activation if the user still needs to complete the WCPayments setup. Don't show it if the user already created a WCPayments account. 

**Tasks:**

- [ ] Change check from 7 days to 3 days in `WC_Payments_Admin::add_menu_notification_badge()` ([source](https://github.com/Automattic/woocommerce-payments/blob/fba91fd6d66fefdc6d1ddd67a00714ea6b2215a2/includes/admin/class-wc-payments-admin.php#L598))
- [ ] Change associated unit tests

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #2837 

#### Changes proposed in this Pull Request

Updates the red badge to display after 3 days instead of 7.

Screen Shot 2021-10-13 at 5 02 38 PM
#### Testing instructions

1. Spin up a new site (or delete `wcpay_menu_badge_hidden`)
2. Do not connect stripe
3. Wait 3 days (or modify `wcpay_activation_timestamp` to be a timestamp >3 days ago)
4. Check that the red badge is shown next to "Payments" in the menu

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)

### CODE REVIEW

File: tests/unit/admin/test-class-wc-payments-admin.php [hunk 1]
```diff
@@ -108,13 +108,13 @@ public function test_it_does_not_render_payments_badge_if_stripe_is_connected()
 		$this->assertArrayNotHasKey( 'wc-admin&path=/payments/connect', $item_names_by_urls );
 	}
 
-	public function test_it_renders_payments_badge_if_activation_date_is_older_than_7_days_and_stripe_is_not_connected() {
+	public function test_it_renders_payments_badge_if_activation_date_is_older_than_3_days_and_stripe_is_not_connected() {
```
Comment (louwie17) [inline tests/unit/admin/test-class-wc-payments-admin.php:111]: The test below this one `test_it_does_not_render_payments_badge_if_activation_date_is_less_than_7_days` will need fixing to, it is failing at the moment

**Review by louwie17 — COMMENTED** [review]
Nice simple change and tested well! There is one PHP test that is failing and a lot of the checks are failing, I assume the latter will just require a rebase.
Will approve after the above two items are resolved.

File: tests/unit/admin/test-class-wc-payments-admin.php (hunk 1 above)
Comment (joshuatf) [reply tests/unit/admin/test-class-wc-payments-admin.php:111]: Thanks, @louwie17 - completely missed that one.  Ready for re-review.

**Review by louwie17 — APPROVED** [review]
Looks good @joshuatf, it's ready to 🚢

**Pull request merged**